source/gui/addonStoreGui/controls/storeDialog.py

```diff
@@ -155,6 +155,18 @@ def _createFilterControls(self, filterCtrlHelper: guiHelper.BoxSizerHelper) -> N
 		filterCtrlsLine1.sizer.AddSpacer(FILTER_MARGIN_PADDING)
 		filterCtrlHelper.addItem(filterCtrlsLine1.sizer, flag=wx.EXPAND, proportion=1)
 
+		self.columnFilterCtrl = cast(
+			wx.Choice,
+			filterCtrlsLine0.addLabeledControl(
+				# Translators: The label of a selection field to filter the list of add-ons in the add-on store dialog.
+				labelText=pgettext("addonStore", "Sort by colu&mn:"),
+				wxCtrlClass=wx.Choice,
+				choices=[c.displayString for c in self._storeVM.listVM.presentedFields],
```

Comment:
I think this should include ascending vs descending for all values

Reply:
A checkbox labelled Descending order has been added after the combo box.

Reply:
I don't think a separate checkbox should be used, this seems very unusual to me. Can you point to a web example that does this? My experience is that the combobox should include separate items for ascending vs descending.

Examples: 
- https://ux.stackexchange.com/questions/43386/sorting-tabular-data-using-a-dropdown-menu
- https://www.officeworks.com.au/shop/officeworks/search?q=chair&view=grid&page=1&sortBy=bestmatch